aplicacion/backend/controllers/experiencias/conseguirExperienciaID.js

```diff
@@ -1,18 +1,44 @@
+const conexionMysql = require("../../DB/conexionMysql");
+
+
 /**
  * Selecciona de la tabla de experiencias un único registro especificando su ID. ❌ 
  * @param {} req 
  * @param {*} res 
  * @param {*} next 
  */
 async function conseguirExperienciaID(req, res, next) {
+    let conexion;
     try {
-        res.statusCode = 200;
+        conexion = await conexionMysql();
+
+        const {id} = req.params;
+
+        const [valor] = await conexion.query(`
+            SELECT id FROM experiencias WHERE id=?
+        `, [id])
+        
+
+        if (valor.length === 0){
+            const err = new Error("No existe ninguna experiencia con ese ID")
+            err.httpStatus = 404;
+            throw err;
+        }
+
+        const [result] = await conexion.query(`
+        SELECT * FROM experiencias WHERE id=?
+    `, [id])
+    
+
+
```

Comment:
Aquí falta establecer el código de estado de respuesta correcta, como hiciste con el otro controlador
```suggestion
res.statusCode = 200;
```